edb/server/protocol/execute.pyx

```diff
@@ -182,6 +182,11 @@ async def execute(
                 await dbv.apply_config_ops(be_conn, config_ops)
 
     except Exception as ex:
+        # If we made schema changes, include the new schema in the
+        # exception so that it can be used when interpreting.
+        if query_unit.user_schema:
+            ex._user_schema = query_unit.user_schema
```

Comment:
Given that we only need this for `pgerror.BackendError`, should we check for that before mutating it?

Reply:
I feel kind-of uneasy about attempting to `setattr` something on a random exception object.

Reply:
Yeah, that's probably a good idea